src/v2/view-builder/views/IdentifierView.js

```diff
@@ -1,34 +1,30 @@
-import { View, loc, createButton } from 'okta';
+import { loc } from 'okta';
 import BaseView from '../internals/BaseView';
 import BaseForm from '../internals/BaseForm';
 import BaseFooter from '../internals/BaseFooter';
+import signInWithIdps from './signin/SignInWithIdps';
```

Comment:
nit:
maybe use `identify` instead of `signin` ?

Reply:
I saw that most of the content in the view was referring to sign in and had classnames set as sign in as well, hence I preferred this name.